frontend/components/ActivityDetails/InstallDetails/SoftwareInstallDetails/SoftwareInstallDetails.tsx

```diff
@@ -121,18 +122,37 @@ export const SoftwareInstallDetails = ({
 
   if (isLoading) {
     return <Spinner />;
-  } else if (isError && error?.status === 404) {
-    return (
-      <DataError
-        description="Install details are no longer available for this activity."
-        excludeIssueLink
-      />
-    );
-  } else if (isError) {
-    return <DataError description="Close this modal and try again." />;
-  } else if (!result) {
+  }
+
+  if (isError) {
+    if (error?.status === 404) {
```

Comment:
Mind inlining the description? Setting the description const ahead of the DataError makes it look like it might impact the branch where deviceAuthToken is specified even though description is unused there.

Reply:
!00% removed it from there per design but didn't adjust code, tyty